lib/vsc/install/shared_setup.py

```diff
@@ -1353,9 +1358,6 @@ def action_target(package, *args, **kwargs):
     """
     This main is the setup.py for vsc-install
     """
-    install_requires = [
```

Comment:
except that that PR doesn't actually adds any restrictions. it'll be quite a challenge to get < 0.6 :smile: